code/modules/power/solar.dm

```diff
@@ -420,6 +420,11 @@
 
 		if(adir != ndir)
 			SPAWN(10+rand(0,15))
-				adir = (360+adir+clamp(ndir-adir,-10,10))%360
-				UpdateIcon()
+				var/old_adir = adir
+				var/max_move = rand(8, 12)
+				adir = (360 + adir + clamp((180 - (540 - ndir + adir) % 360), -max_move, max_move)) % 360
+				if(adir != old_adir)
+					use_power(power_usage)
```

Comment:
```suggestion

```
`/obj/machinery/power/solar/owl_cheat`

Reply:
Ah fair point, power usage reverted - it never did draw power did it.